### ISSUE TITLE
Time of units start training

### ISSUE THREAD

**Issue by hypatiasf** [Blizzard/s2protocol#81]
Which file in the archive can we find when a unit start training? (not the time finish training)

I found something in file 'replay.tracker.events' which is close to the aim I think, they're about:

- a unit finished training (event 'NNet.Replay.Tracker.SUnitBornEvent') 
- a structure start constructing (event 'NNet.Replay.Tracker.SUnitInitEvent') 

but none of them are suitable.

**Comment by amartin916** [comment]
I don't believe the info is in the replay file, how I go about it is I keep track of the build times for the units so the finished time - build time.

**Comment by hypatiasf** [comment]
Thank you for your advice, I think that would be useful in most conditions.  But ... what if a protoss player used a chrono boost?

**Comment by amartin916** [comment]
You would have to account for it, by finding what building the chrono was used on, and also linking the unit to the building. "Might be able to link the unit back to the building with m_creatorUnitTagIndex and m_creatorUnitTagRecycle I have never looked. I know that for zerg everything links back to the larva".

I personally don't worry about the chrono but I also lean more on the complete time than the starting. And where I lean on the started time you would be hard press to notice the difference since I only record things in 7 second intervals

**Comment by hypatiasf** [comment]
@amartin916  It should be the solution, thank you!
